README.md

```diff
@@ -57,25 +57,6 @@
 
 ## 📦 GH-pages deployment
 
-1. **Basic name.github.io/deriv-com/**
```

Comment:
we are adding gh-pages, why do we remove readme for gh-pages here?

actually, gh-pages is only a fallback for development, in case vercel or other test link deployers are down

Reply:
We're removing the duplicated paragraph. We have two identical paragraphs for `gh-pages`.
See the image below:

![duplicate](https://user-images.githubusercontent.com/59390029/88771544-a31a5b80-d194-11ea-8eaf-5aa12584f1cf.png)

Reply:
i see, thank you 🙏